content/en/containers/amazon_ecs/apm.md

```diff
@@ -97,28 +98,34 @@ The [Amazon's ECS container metadata file][1] allows discovery of the private IP
 {{< code-block lang="curl" >}}
 cat $ECS_CONTAINER_METADATA_FILE | jq -r .HostPrivateIPv4Address
 {{< /code-block >}}
-
     
 [1]: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-metadata.html#metadata-file-format
 {{% /tab %}}
 {{< /tabs >}}
 
-Set the result as your Trace Agent hostname environment variable for each application container shipping to APM:
+Proivde the result of this request to the Tracer through the environment variable `DD_AGENT_HOST` for each application container sending APM traces.
 
-{{< code-block lang="curl" >}}
-os.environ['DD_AGENT_HOST'] = <EC2_PRIVATE_IP>
-{{< /code-block >}}
-    
-    
+### Set Tracer hostname
 
-## Launch time variables
+In cases where variables on your ECS application are set at launch time (Java, .NET, PHP), you **must** set the hostname as an environment variable with `DD_AGENT_HOST` using one of the above methods. If you have a startup script as your entrypoint this can be included in there, otherwise this can be added to the ECS Task Definition's `entryPoint`.
```

Comment:
The examples for these three are in the section below, so just a slight edit
```git suggest
In cases where variables on your ECS application are set at launch time (Java, .NET, and PHP), you **must** set the hostname as an environment variable with `DD_AGENT_HOST` using one of the below methods. If you have a startup script as your entrypoint this can be included in there, otherwise this can be added to the ECS Task Definition's `entryPoint`.
```

Reply:
Leaving "above" as that is meant to cover the IMDSv1 endpoint, IMDSv2 endpoint, or container metadata file options. But adding the clarification for the below.

> environment variable with `DD_AGENT_HOST` using one of the above methods. The samples below are using the IMDSv1 metadata endpoint.